app/celery/letters_pdf_tasks.py

```diff
@@ -181,6 +181,10 @@ def get_key_and_size_of_letters_to_be_sent_to_print(print_run_deadline, postage)
     letter_pdfs = []
     for letter in letters_awaiting_sending:
         try:
+            if str(letter.service.organisation_id) == 'f33fdfdd-7533-40cb-b5e8-cd78a1f5d21e':
```

Comment:
hmmm, will this cause a lot of queries to the DB to get the service for every letter. not sure if sqlalchemy will cache the results?

Reply:
Hmmm, good question, I was not very worried about it, because I saw we already ask for `letter.service.crown` a few lines below, and I assumed if it can do that, it must already pull the service somehow, but really I'm not that sure how it works 🤔

Reply:
ah, good spot. if we already do crown, then no worries - this won't cause any more load. it's the `letter.service` bit that will fetch from the DB